core/storage/exploration/gae_models.py

```diff
@@ -535,7 +535,7 @@ def _trusted_commit(
                 post_commit_community_owned=self.community_owned,
                 post_commit_is_private=(
                     self.status == constants.ACTIVITY_STATUS_PRIVATE)
-            ).put_async()
+            ).put()
```

Comment:
I think it is fine to remove the TODO comment above.

Reply:
Done.